system_tests/test_cli_package.py

```diff
@@ -241,54 +241,41 @@ def bootstrap_manager(self):
         if self.is_install_plugins():
             install_plugins = '--install-plugins'
         out = self._execute_command(
-            'bootstrap -p {0} -i {1} {2}'.format(
-                self.test_manager_blueprint_path,
-                self.remote_bootstrap_inputs_path,
+            """bootstrap -p {0} -i "{1}" {2}""".format(
```

Comment:
Yeah, so it was a single quote, but windows is being lame and doesn't get it as it should...
And the triple quotes is just for readability ( I could've used the escape char for the inner double quotes).